.github/workflows/build.yml

```diff
@@ -1,11 +1,10 @@
-name: Warp CI
+name: Behaviour tests
```

Comment:
The workflow runs `yarn test` not `yarn test:behaviour` what is the difference between them? I thought yarn test runs all tests and `yarn test:whatever` only specific

Reply:
I think test just runs behaviour, because it doesn't invoke semantic tests we have a different script for them.
And the cli tests are explicitly excluded from them because @RohitRanjangit wanted them to be different.
@RohitRanjangit thoughts, let's resolve this quick so that we can merge

Reply:
As of now , the name should be `Warp CI` in my opinion. I agree with @piwonskp that `yarn test` does bunch of things beside running behv tests.